services/blockchain-app-registry/shared/metadata.js

```diff
@@ -212,7 +212,7 @@ const getBlockchainAppsMetadata = async (params) => {
 		{ concurrency: blockchainAppsMetadata.data.length },
 	);
 
-	const [{ total }] = await applicationMetadataTable.rawQuery(`SELECT COUNT(chainName) as total from ${applicationMetadataIndexSchema.tableName}`);
+	const total = await applicationMetadataTable.count(params, ['chainName']);
```

Comment:
Can you please verify if `count` method accepts second params?

Reply:
Yes, you are right. It accepts only a single parameter. Will update.